dbs/mongo/queries/getAnswerData.js

```diff
@@ -0,0 +1,17 @@
+const {Answers, Answer_Photos} = require('../models.js');
+const {formatAnswerResponse} = require('../controllers/answerResponse.js')
+
+async function getAnswerData(q_id, page, count) {
+
+  let answers = await Answers.find({question_id: q_id, reported:0}, {_id: 0}).limit(count);
+
+  let answerPhotos = await Promise.all(answers.map(async (answer) => {
```

Comment:
Is it necessary to use Promise.all when you are using async - await? And where those promises get resolved / rejected?